remi/rutil.js

```diff
@@ -49,26 +38,17 @@ function printTimeStamp (time) {
  * @return {string} current time stamp in readable format
  */
 function getTimeStamp () {
-  const today = new Date()
-  const date = (today.getUTCMonth() + 1) + '/' + today.getUTCDay() + '/' + today.getFullYear()
-  const time = today.getUTCHours() + ':' + today.getUTCMinutes() + ':' + today.getUTCSeconds()
-  return `${date} ${time}`
+  const {getUTCMonth, getUTCDay, getFullYear, getUTCHours, getUTCMinutes, getUTCSeconds} = new Date()
```

Comment:
I'm getting a strange error with getUTCMonth, for some reason it's not recognized as a Date object? Are we calling it wrong here or missing a library or something?

Output when running:
```
(node:18046) UnhandledPromiseRejectionWarning: TypeError: this is not a Date object.
    at getUTCMonth (<anonymous>)
    at getTimeStamp (/home/androo1/Workspace/REMi/remi/rutil.js:42:13)
    at Object.err (/home/androo1/Workspace/REMi/remi/rutil.js:92:19)
    at Object.connectDB (/home/androo1/Workspace/REMi/remi/mongoUsers.js:77:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:18046) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:18046) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:18046) UnhandledPromiseRejectionWarning: TypeError: this is not a Date object.
    at getUTCMonth (<anonymous>)
    at getTimeStamp (/home/androo1/Workspace/REMi/remi/rutil.js:42:13)
    at Object.log (/home/androo1/Workspace/REMi/remi/rutil.js:58:19)
    at Client.<anonymous> (/home/androo1/Workspace/REMi/remi/remi.js:36:9)
    at Client.emit (events.js:315:20)
    at WebSocketManager.triggerClientReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:433:17)
    at WebSocketManager.checkShardsReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:417:10)
    at WebSocketShard.<anonymous> (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketManager.js:199:14)
    at WebSocketShard.emit (events.js:315:20)
    at WebSocketShard.checkReady (/home/androo1/Workspace/REMi/node_modules/discord.js/src/client/websocket/WebSocketShard.js:467:12)
(node:18046) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

Reply:
oh I get it, the js date object implementation is shit code. Replace const {getUTCMonth ... etc } with const date = new Date() and then add date. to the front of each method call in the format string